src/pages/quizzes.tsx

```diff
@@ -0,0 +1,281 @@
+    header: <Translation id="test-your-knowledge" />,
+    subtitle: <Translation id="quizzes-subtitle" />,
+    image: getImage(data.doge)!,
+    alt: t("quizzes-title"),
+  }
+
+  const contextState = {
+    status: quizStatus,
+    next: nextQuiz,
+    score: userStats.score,
+    average: userStats.average,
+    completed: userStats.completed,
+  }
+
+  return (
+    <Box>
+      <PageMetadata
+        title={t("quizzes-title")}
+        description={t("quizzes-subtitle")}
+      />
+      <HeroContainer>
+        <Hero content={heroContent} isReverse />
+      </HeroContainer>
+
+      <QuizzesHubContext.Provider value={contextState}>
+        <QuizzesModal isOpen={isModalOpen} setIsOpen={setModalOpen}>
+          <QuizWidget
+            quizKey={currentQuiz}
+            nextHandler={setCurrentQuiz}
+            statusHandler={setQuizStatus}
+            setUserStats={setUserStats}
+            isStandaloneQuiz={false}
+          />
+        </QuizzesModal>
+
+        <Box px={{ base: 0, lg: 8 }} py={{ base: 0, lg: 4 }} mb={12}>
+          <Flex
+            direction={{ base: "column", lg: "row" }}
+            alignItems="start"
+            gap={{ base: 0, lg: 20 }}
+          >
+            {/* quizzes list */}
+            <Box flex={1} order={{ base: 2, lg: 1 }}>
+              <Box px={{ base: 8, lg: 0 }}>
+                <Heading
+                  fontSize={{ base: "1.75rem", lg: "2rem" }}
+                  color="body"
+                >
+                  <Translation id="basics" />
+                </Heading>
+
+                <Text mb={8} color="body">
+                  <Translation id="basics-description" />
+                </Text>
+
+                <QuizzesList
+                  content={ethereumBasicsQuizzes}
+                  quizHandler={setCurrentQuiz}
+                  nextHandler={setNextQuiz}
+                  modalHandler={setModalOpen}
+                />
+              </Box>
+
+              <Box px={{ base: 8, lg: 0 }} mb={10}>
+                <Heading
+                  fontSize={{ base: "1.75rem", lg: "2rem" }}
+                  color="body"
+                >
+                  <Translation id="using-ethereum" />
+                </Heading>
+
+                <Text mb={8} color="body">
+                  <Translation id="using-ethereum-description" />
+                </Text>
+
+                <QuizzesList
+                  content={usingEthereumQuizzes}
+                  quizHandler={setCurrentQuiz}
+                  nextHandler={setNextQuiz}
+                  modalHandler={setModalOpen}
+                />
+              </Box>
+
+              <Flex
+                direction={{ base: "column", lg: "row" }}
+                justifyContent="space-between"
+                alignItems="center"
+                bg="ednBackground"
+                borderRadius={{ base: "none", lg: "lg" }}
+                border="none"
+                p={{ base: 8, lg: 12 }}
```

Comment:
```suggestion
                p={4}
```

I think the designs are 16px padding at all screen sizes, though I agree it does look more balanced with a little more padding on smaller screen sizes. cc: @konopkja 

<img width="627" alt="Screenshot 2023-05-29 at 10 34 53" src="https://github.com/ethereum/ethereum-org-website/assets/62268199/b4ab0543-6d72-4571-92f5-0b10cf2e23d8">
<img width="727" alt="Screenshot 2023-05-29 at 10 35 05" src="https://github.com/ethereum/ethereum-org-website/assets/62268199/f7abe3c9-858f-42d0-b2a1-21582f6b162c">

<img width="776" alt="Screenshot 2023-05-29 at 10 37 26" src="https://github.com/ethereum/ethereum-org-website/assets/62268199/1dac3e5b-98b3-472a-a9ad-c81ec04cd71a">
<img width="735" alt="Screenshot 2023-05-29 at 10 37 34" src="https://github.com/ethereum/ethereum-org-website/assets/62268199/45ba3172-6046-4dbf-831c-4e56d815ad32">

Reply:
a little but lost here, but generally i would advise to have more empty space as we go lower with the device screen size, ultimately would follow guidance from @nloureiro on paddings tho

Reply:
what padding are we talking about?

Reply:
got it, its about the padding on the "want to see more quizzes..." box


we should use 16px/1rem for the spacing

![Screen Shot 2023-05-31 03 08 01 PM](https://github.com/ethereum/ethereum-org-website/assets/1120748/156635d8-c07b-4e6b-94b0-137034f2dbc3)

Reply:
Updated to use 1rem padding for the cards